network-api/networkapi/wagtailpages/wagtail_hooks.py

```diff
@@ -633,3 +634,20 @@ def register_donate_banner_chooser_viewset():
         model=wagtailpages_models.BuyersGuideProductCategory,
         url_prefix="wagtailpages/buyersguideproductcategory",
     )
+
+
+# --------------------------------------------------------------------------------------
+# Custom Wagtail A/B Testing Events:
+# --------------------------------------------------------------------------------------
+
+
+class DonateBannerLinkClick(BaseEvent):
+    name = "Donate Banner Link Click"
+    requires_page = False  # Set to False to create a "Global" event type that could be reached on any page
+
+
+@hooks.register("register_ab_testing_event_types")
+def register_donate_banner_link_click_event_type():
+    return {
+        "donate-banner-link-click": DonateBannerLinkClick,
+    }
```

Comment:
For more details on adding custom Wagtail A/B testing events, refer to the [Wagtail A/B testing docs](https://github.com/wagtail-nest/wagtail-ab-testing?tab=readme-ov-file#implementing-custom-goal-event-types).